tests/t537-operator.c

```diff
@@ -87,11 +90,17 @@ int main(int argc, char **argv) {
   // Assemble diagonal
   CeedVectorCreate(ceed, num_comp * num_comp * num_dofs, &assembled);
   CeedOperatorLinearAssemblePointBlockDiagonal(op_mass, assembled, CEED_REQUEST_IMMEDIATE);
+  CeedOperatorLinearAssemblePointBlockDiagonalSymbolic(op_mass, &num_entries, &rows, &cols);
 
-  // Manually assemble diagonal
+  // Manually assemble point-block diagonal
   CeedVectorCreate(ceed, num_comp * num_dofs, &u);
   CeedVectorSetValue(u, 0.0);
   CeedVectorCreate(ceed, num_comp * num_dofs, &v);
+  for (int i = 0; i < num_comp * num_dofs; i++) {
+    for (int j = 0; j < num_comp * num_dofs; j++) {
```

Comment:
```suggestion
  for (CeedInt i = 0; i < num_comp * num_dofs; i++) {
    for (CeedInt j = 0; j < num_comp * num_dofs; j++) {
```

Reply:
Currently every `t*` file uses `CeedInt`. We can change them all to use `int` instead, but that should be a separate PR if we want to do that.

Reply:
Whoops, I answered this through my phone and thought this was a fluids PR, not for the library. I'll accept the changes when I get back to my computer.